src/sidebar/components/annotation-user.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const propTypes = require('prop-types');
+const { createElement } = require('preact');
+
+const { isThirdPartyUser, username } = require('../util/account-id');
+const { withServices } = require('../util/service-context');
+
+/**
+ * Display information about an annotation's user. Link to the user's
+ * activity if it is a first-party user or `settings.usernameUrl` is present.
+ */
+function AnnotationUser({ annotation, features, serviceUrl, settings }) {
+  const user = annotation.user;
+  const isFirstPartyUser = !isThirdPartyUser(user.userid, settings.authDomain);
```

Comment:
`annotation.user` is a string containing an account ID. It does not have a `userid` property.